dbt/models/tax/schema.yml

```diff
@@ -45,8 +45,7 @@ sources:
                     where:
                       CAST(year AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
                   meta:
-                    category: class_mismatch_or_issue
-                    description: class code must be valid
+                    description: tax_code_num should match legdat.taxdist
```

Comment:
So, we actually know and expect this test to fail for the time being, as the `taxdist` values in `legdat` are out of date. We don't really have a handler for the "this is failing now but needs to be fixed in the future" case. Perhaps we should make this into a warning rather than an explicit error. Thoughts @jeancochrane?

Reply:
That makes sense to me @dfsnow! Do we know which `taxdist` values are out of date, and when we expect them to be up to date? If not, I think a naive warning on the failing row count is a good start, but if we do know how many and when we expect to be fixed it could be nice to add some additional filtering logic so that it starts failing automatically if one of those assumptions is broken.

Reply:
If I recall right all values after 2021 are not kept up-to-date and are supposed to be fixed by the Clerk once they come on board iasWorld. The tax codes don't change super frequently so many of them may still match fine. I would start with a naive warning for now.

Reply:
@dfsnow 
Do we want the warning if the errors are >0 or if they are > 'current value'?

Reply:
@dfsnow Also, as the query is currently constructed, there are mismatched values, but none after 2021. Below is a table with the years where errors are populated (after removing the year clause).

2006
2007
2008
2009
2010
2011
2012
2014
2017
2018
2019
2020
2021

Reply:
Whoops, maybe I had that backwards, it could be that values **before** 2021 are not up-to-date. Either way, let's throw a warning for errors>0.

Reply:
Done